ui/vue.config.js

```diff
@@ -1,4 +1,5 @@
 module.exports = {
+  transpileDependencies: ['alcaeus'],
```

Comment:
Is this necessary? I tried and Alcaeus "just worked"

Reply:
I didn't try without it. Just ported that from the old project. Will try